docs/src/rfc/text/0020-remove-atomics.md

```diff
@@ -0,0 +1,201 @@
+# RFC: `Remove Atomic from Patina`
+

```

Comment:
Nit on process: Because this is a revisement to RFC 18, albeit a substantial one, I think that we wouldn’t want to delete 18 and create a new one, but still do this as an amendment to 18.

Reply:
I'm not renaming it to 20 because of the revision; but another 18 snuck in and I completed this one without noticing the duplication; so the rename is just to remove the duplication.

Reply:
(Duplicated with https://github.com/OpenDevicePartnership/patina/blob/main/docs/src/rfc/text/0018-system-tests.md)